doc/flame_fire_atlas/fire_atlas.md

```diff
@@ -0,0 +1,83 @@
+# Flame fire atlas
+
+Flame fire atlas is a texture atlas lib for Flame. Using `Flame fire atlas` one can access images
+and animations stored in a `.fa` texture atlas by referring to them by their named keys.
+
+
+## FireAtlas
```

Comment:
There should be some instructions for how to use FireAtlas here

Reply:
added